src/actions/rootActions.js

```diff
@@ -4,15 +4,21 @@ import {
   APP_LAUNCHED
 } from './types'
 
-export const showNotification = (payload = { message: 'Empty message' }) => {
-  const newPayload =
-    typeof payload === 'string' ? { message: payload } : payload
+export const showNotification = (payload = { title: 'Empty message' }) => {
+  const newPayload = typeof payload === 'string' ? { title: payload } : payload
+
+  const notificationId = Date.now()
+
+  newPayload.variant = newPayload.variant || 'info'
+  newPayload.dismissAfter = newPayload.dismissAfter || 4000
 
   return {
     type: SHOW_NOTIFICATION,
     payload: {
       ...newPayload,
-      key: new Date().getTime()
+      solidFill: true,
+      key: notificationId,
+      id: notificationId
```

Comment:
1. Why have both `key` and `id`? (Data duplication)
2. `solidFill`, `dismissAfter` and other defaults should be applied in the reducer.
```js
case SHOW_NOTIFICATION:
return {
   dismissAfter: 4000,
   solidFill: true,
   variant: 'info',
   ...action.payload
}
```